cmd/containerboot/main.go

```diff
@@ -201,7 +204,9 @@ func main() {
 			log.Fatalf("error setting up for running on Kubernetes: %v", err)
 		}
 	}
-
+	if cfg.App != "" {
+		hostinfo.SetApp(cfg.App)
```

Comment:
This won't work here as this is in containerboot and you need to set this in tailscaled.

Reply:
I've fixed this and tested that it now is actually set as expected